test/reference-image

```diff
@@ -0,0 +1 @@
+fedora-35
```

Comment:
This really belongs into the previous commit (testlib bump), not the "add rsync ability" one.

Reply:
Correct! Will fix.